ui/components/app/advanced-gas-fee-popover/advanced-gas-fee-save/advanced-gas-fee-save.js

```diff
@@ -9,22 +10,51 @@ import I18nValue from '../../../ui/i18n-value';
 import { useAdvancedGasFeePopoverContext } from '../context';
 import { decGWEIToHexWEI } from '../../../../../shared/modules/conversion.utils';
 
+const getDifference = (num1, num2) =>
+  new BigNumber(num1, 10).minus(new BigNumber(num2, 10)).toNumber();
+
 const AdvancedGasFeeSaveButton = () => {
   const { closeAllModals } = useTransactionModalContext();
-  const { updateTransaction } = useGasFeeContext();
+  const { captureTransactionMetrics, updateTransaction } = useGasFeeContext();
   const {
     gasLimit,
     hasErrors,
+    initialGasValues,
     maxFeePerGas,
     maxPriorityFeePerGas,
   } = useAdvancedGasFeePopoverContext();
 
   const onSave = () => {
-    updateTransaction({
-      estimateUsed: PRIORITY_LEVELS.CUSTOM,
+    const gasValues = {
       maxFeePerGas: decGWEIToHexWEI(maxFeePerGas),
       maxPriorityFeePerGas: decGWEIToHexWEI(maxPriorityFeePerGas),
       gasLimit,
+    };
+    updateTransaction({
+      estimateUsed: PRIORITY_LEVELS.CUSTOM,
+      ...gasValues,
+    });
+    captureTransactionMetrics({
+      action: 'Advance gas fee modal',
```

Comment:
Advance -> Advanced? Same for the other instances.

Reply:
fixed 👍